pkg/controller/account/iam.go

```diff
@@ -71,16 +71,10 @@ func (r *ReconcileAccount) CreateSecret(reqLogger logr.Logger, account *awsv1alp
 
 // getStsCredentials returns STS credentials for the specified account ARN
 // Takes a logger, an awsClient, a role name to assume, and the target AWS account ID
-func getStsCredentials(reqLogger logr.Logger, client awsclient.Client, iamRoleName string, awsAccountID string) (*sts.AssumeRoleOutput, error) {
-	// Use the role session name to uniquely identify a session when the same role
-	// is assumed by different principals or for different reasons.
-	var roleSessionName = "awsAccountOperator"
+func getStsCredentials(reqLogger logr.Logger, client awsclient.Client, roleArn string, roleSessionName string) (*sts.AssumeRoleOutput, error) {
```

Comment:
Maybe this should be named `getSTSCredentials` to match Go convention and `validateSTSClaim` etc.

Reply:
Done